requirements.txt

```diff
@@ -6,7 +6,7 @@ django-smoketest==1.1.2
 djangorestframework==3.12.4
 pytz==2021.1
 raven==6.10.0
-six==1.15.0
+six==1.16.0
```

Comment:
This seems to be unused. Do we have to keep it?

```suggestion
```

Reply:
we don't use it ourselves, but a lot of dependencies use it.